src/js/extensions/paste.js

```diff
@@ -60,7 +60,10 @@
             [new RegExp(/<\/?o:[a-z]*>/gi), ''],
 
             // Microsoft Word adds some special elements around list items
-            [new RegExp(/<!\[if !supportLists\]>(((?!<!).)*)<!\[endif]\>/gi), '$1']
+            [new RegExp(/<!\[if !supportLists\]>(((?!<!).)*)<!\[endif]\>/gi), '$1'],
+
+            // remove font tag
+            [new RegExp(/<font[^><]*>|<.font[^><]*>/gi), '']
```

Comment:
Are there any attributes on the font tags that help identify it as word / open office?  The problem here is that when copying and pasting from various sources there can be valid `<font>` tags to maintain the font type, size, or color, so I'm not sure we'd want to always remove all `<font>` tags when pasting...